templates/component/_sideNavbar.html.twig

```diff
@@ -1,13 +1,29 @@
 <div id="sidebar" class="bg-primary show shadow-lg sticky-md-top">
     <div class="list-group text-sm-start min-vh-100">
         <ul class="list-unstyled text-white text-center sticky-md-top">
-            <button type="button" class="btn text-reset d-md-none" data-bs-dismiss="offcanvas" aria-label="Close"><i
-                    class="bi bi-x-circle-fill fs-4 text-white"></i></button>
+            <button type="button" class="btn text-reset d-md-none" data-bs-dismiss="offcanvas" aria-label="Close">
+                <i class="bi bi-x-circle-fill fs-4 text-white"></i></button>
             <a href="/" class="nav-link">
                 <li class="m-3 rounded fs-5">
                     <i class="bi bi-house-fill"></i> Accueil
+
                 </li>
             </a>
+
+                <div class="overflow-y-auto">
+                    <button class="btn btn-primary" type="button" data-bs-toggle="collapse" data-bs-target="#perimeterList" aria-expanded="false" 
+                    aria-controls="perimeterList">
+                        Catégories
+                    </button>
+                    <div class="collapse" id="perimeterList">
+                        {% for perimeter in perimeters.getDistinctPerimeters %}
+                        <div class="card"><a class="dropdown-item" href="#">{{perimeter.perimeter}}</a>
```

Comment:
Better ! But still too much. You don't need to get here full Idea objects. getDistinctPerimeters should return an array of string, not an array of ideas